### PR TITLE
ENYO-2411 IntegerPicker does not update range at creation

### PR DESCRIPTION
**Pull request by dragongears**
Enyo-DCO-1.1-Signed-off-by: Art Dahm art.dahm@lge.com

### CODE REVIEW

File: src/IntegerPicker/IntegerPicker.js [hunk 1]
```diff
@@ -247,7 +247,7 @@ module.exports = kind(
 	*/
 	create: function () {
 		Control.prototype.create.apply(this, arguments);
-		this.verifyValue();
```
Comment (Djspaceg) [inline src/IntegerPicker/IntegerPicker.js]: We no longer need to run `verifyValue`? That sets the value and allows the control to already be set to the value without animating.

Comment (dragongears) [inline src/IntegerPicker/IntegerPicker.js]: verifyValue() gets called inside rangeChanged(), so I removed it.

Comment (Djspaceg) [inline src/IntegerPicker/IntegerPicker.js]: Super. Didn't see that.

File: src/IntegerPicker/IntegerPicker.js [hunk 2]
```diff
@@ -247,7 +247,7 @@ module.exports = kind(
...
+		this.rangeChanged();
```
Comment (clinuz) [inline src/IntegerPicker/IntegerPicker.js]: Calling this in `create` causes the `onChange` event to be fired during create. Which is something we REALLY don't want. It was being avoided in the current implementation and may cause problems by changing this.

**Pull request merged**